webpack.config.js

```diff
@@ -62,6 +67,26 @@ const commonConfig = {
     },
 };
 
+const electronConfig = {
+    ...commonConfig,
+    entry: electronEntryFiles,
+    name: 'electron',
+    mode: 'development',
+    output: {
+        path: path.join(__dirname, 'extension/electronBundle'),
+        filename: '[name].bundle.js',
+    },
+    node: {
+        setImmediate: false,
```

Comment:
A possible alternate:
```suggestion
        ...commonConfig.node,
```
(instead of manually copy/pasting the node settings we want to share in source and risking unintentional divergence, copy all the common node settings at run time and extend?)